src/server/utils/scripts/email.ts

```diff
@@ -0,0 +1,31 @@
+import { createTransport } from 'nodemailer';
+import { join } from 'path';
+

```

Comment:
Can I know what is the purpose of this script?

Reply:
Hahahaha nothing. Just a demo 😂